bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/niri/Niri.java

```diff
@@ -797,20 +793,6 @@ public ROIDescription(ROIDescription roid) {
         }
 
         /**
-         * Return the x start pixel.
-         */
-        public int getXStart() {
-            return _xStart;
-        }
-
-        /**
-         * Return the y start pixel.
-         */
-        public int getYStart() {
-            return _yStart;
-        }
-
-        /**
```

Comment:
I should have asked... I removed these 2 accessors as they aren't being used in the OCS code tree. However, now  I wonder if there is any other software that is relying on these things? LCH? ITC? Or maybe something using reflection  (oh my, I hope not!) that could be accessing these? Maybe I should just leave these in to be safe, but it's kind of annoying to leave stuff in not knowing if it is really necessary. It's kind of strange that this class has these fields defined but nobody seems to be using them - I'm of the idea of removing as much code as we can that we don't need. What do you guys do normally in these situations?